File: python/unagent_lint/rules.py
def not_an_agent(content, file):
    if "acts on your behalf" in content.lower():
        return {
            "rule": "ID-001",
            "severity": "FAIL",
            "message": "UNAGENT must not be described as an autonomous agent.",
            "file": file,
        }


RULES = [not_an_agent]

File: node/lint/rules.js
module.exports = [
  {
    id: "ID-001",
    severity: "FAIL",
    check(content, file) {
      const lowered = content.toLowerCase();
      if (lowered.includes("acts on your behalf")) {
        return {
          rule: "ID-001",
          severity: "FAIL",
          message: "UNAGENT must not be described as an autonomous agent.",
          file
        };
      }
    }
  }
];
